### ISSUE TITLE
30-seconds-example - PyProj Issue Perhaps

### ISSUE THREAD

**Issue by nygeog** [up42/up42-py#55]
Greetings, this library is great, was able to run `airports-parallel` notebook. But ran into an error here in the `30-seconds-example` notebook. 

I've been running into errors and warnings w/ `pyproj` lately due to espg changes lately in other code. 

When I run:
```python
aoi.crs
```
Result: `{'init': 'epsg:4326', 'no_defs': True}`

#### PyProj Version I'm Running

```python
import pyproj

pyproj.__version__
```

Result: `'1.9.6'`

#### When I run this cell:

```python
# Define the aoi and input parameters of the workflow to run it.

input_parameters = workflow.construct_parameters(geometry=aoi, 
                                                 geometry_operation="bbox", 
                                                 start_date="2018-01-01",
                                                 end_date="2020-12-31",
                                                 limit=1)
input_parameters["sobloo-s2-l1c-aoiclipped:1"].update({"max_cloud_cover":60})
input_parameters
```
#### Get the Following Error Message: 

```python

AttributeError                            Traceback (most recent call last)
<ipython-input-12-98126c8ec4dd> in <module>
      5                                                  start_date="2018-01-01",
      6                                                  end_date="2020-12-31",
----> 7                                                  limit=1)
      8 input_parameters["sobloo-s2-l1c-aoiclipped:1"].update({"max_cloud_cover":60})
      9 input_parameters

~/opt/anaconda3/envs/dsi/lib/python3.7/site-packages/up42/workflow.py in construct_parameters(self, geometry, geometry_operation, handle_multiple_features, start_date, end_date, limit, scene_ids, order_ids)
    301                 datetime = f"{start_date}T00:00:00Z/{end_date}T00:00:00Z"
    302                 input_parameters[data_block_name]["time"] = datetime
--> 303             aoi_fc = any_vector_to_fc(vector=geometry,)
    304             aoi_feature = fc_to_query_geometry(
    305                 fc=aoi_fc,

~/opt/anaconda3/envs/dsi/lib/python3.7/site-packages/up42/utils.py in any_vector_to_fc(vector, as_dataframe)
    276             df = vector
    277             if df.crs != "epsg:4326":
--> 278                 df = df.to_crs(4326)
    279 
    280     if as_dataframe:

~/opt/anaconda3/envs/dsi/lib/python3.7/site-packages/geopandas/geodataframe.py in to_crs(self, crs, epsg, inplace)
    532         else:
    533             df = self.copy()
--> 534         geom = df.geometry.to_crs(crs=crs, epsg=epsg)
    535         df.geometry = geom
    536         df.crs = geom.crs

~/opt/anaconda3/envs/dsi/lib/python3.7/site-packages/geopandas/geoseries.py in to_crs(self, crs, epsg)
    422         else:
    423             proj_in = pyproj.Proj(self.crs, preserve_units=True)
--> 424             proj_out = pyproj.Proj(crs, preserve_units=True)
    425             project = partial(pyproj.transform, proj_in, proj_out)
    426         result = self.apply(lambda geom: transform(project, geom))

~/opt/anaconda3/envs/dsi/lib/python3.7/site-packages/pyproj/__init__.py in __new__(self, projparams, preserve_units, **kwargs)
    344             projstring = projparams
    345         else: # projparams a dict
--> 346             projstring = _dict2string(projparams)
    347         # make sure units are meters if preserve_units is False.
    348         if not projstring.count('+units=') and not preserve_units:

~/opt/anaconda3/envs/dsi/lib/python3.7/site-packages/pyproj/__init__.py in _dict2string(projparams)
    684     # convert a dict to a proj4 string.
    685     pjargs = []
--> 686     for key,value in projparams.items():
    687         pjargs.append('+'+key+"="+str(value)+' ')
    688     return ''.join(pjargs)

AttributeError: 'int' object has no attribute 'items'


```

I'm happy to take a look at this and fix if you'd like, but thought I'd Submit an Issue as I don't think I can dig into it today. 

Take care.

**Comment by nygeog** [comment]
I think https://github.com/up42/up42-py/blob/master/up42/utils.py#L264 just needs `df = df.to_crs(epsg=4326)`. Will test out fix when I have time.

**Comment by chrieke** [comment]
@nygeog Hi Danny, thank you for your feedback and effort!

The newest version of geopandas (0.7.0) introduced a breaking change for the crs object, the old `{'init': 'epsg:4326', 'no_defs': True}` form is deprecated in favour of `"EPSG:4326"`. To make it a bit clearer we now pinned the geopandas version, cleaned up the code to be more consistent with the crs usage and released a new version 0.8.3.

Please  update to the newest version of up42-py and geopandas via `pip install up42-py --upgrade`. Thanks again for your feedback!

PS: We also split up `create_and_run_job` to `run_job`and `test_job` recently, please have a look at the [changelog](https://github.com/up42/up42-py/blob/master/CHANGELOG.md).
